1.1/spec/06-Part-03.adoc

```diff
@@ -6,7 +6,7 @@ ISO 19125-1, _Geographic information — Simple feature access — Part 1: Commo
 
 OGC 07-036, _Geography Markup Language (GML) Encoding Standard_, Version 3.2.1
 
-RFC 2396, _RFC 2396 - Uniform Resource Identifiers (URI): Generic Syntax_
+RFC 2396, _RFC 2396 - Uniform Resource Identifiers (IRI): Generic Syntax_
```

Comment:
Something is wrong here. Likely a different RFC should be referred to.

Reply:
I've fixed this in the commit below by replacing the reference to the URI spec with a reference to the IRI spec.

I will update the _Normative References_ section of GeoSPARQL in a subsequent PR.

Reply:
The RFC for IRI is https://tools.ietf.org/html/rfc3987

Reply:
That's the reference I've used in _Bibliography_ and have just now overwritten the _Normative References_ URI reference with.

(I suspect some of the confusion with this PR and some others is that there's a spillage of concerns between them as I find an issue, address it but then have to address underlying issues - this PR was really only meant to be about WKT! Well, to fix this I am doing that whole-spec review this week.)